emission/analysis/plotting/geojson/geojson_feature_converter.py

```diff
@@ -285,6 +288,42 @@ def get_geojson_for_timeline(user_id, tl):
 
     return geojson_list    
     
+
+def get_geojson_for_timeline_viz(user_id, tl):
```

Comment:
Why do you need a separate function here? Can you highlight the differences from the existing `get_geojson_for_XXX`? It seems like a copy-paste to me, and I think that copy-pastes are bad and buggy in general.